test/test_scanner.py

```diff
@@ -1012,12 +1029,6 @@ def test_binaries(self, binary, package, version, are_in, not_in):
                         "hostapd",
                         "2.1",
                     ),
-                    (
-                        "https://kojipkgs.fedoraproject.org//packages/python3/3.8.2~rc1/1.fc33/aarch64/",
```

Comment:
As @pdxjohnny says, this shouldn't have been removed.

Reply:
It seems like a duplicate entry. On line 942-946, the same file is being downloaded.